vita3k/renderer/src/gl/renderer.cpp

```diff
@@ -454,6 +454,7 @@ void get_surface_data(GLContext &context, size_t width, size_t height, size_t st
         glReadPixels(0, 0, static_cast<GLsizei>(width), static_cast<GLsizei>(height), GL_RGBA, GL_UNSIGNED_INT_8_8_8_8_REV, pixels);
         break;
     case SCE_GXM_COLOR_FORMAT_U8U8U8U8_ARGB:
+    case SCE_GXM_COLOR_FORMAT_U8U8U8U8_BGRA:
```

Comment:
One is BGRA, the other RGBA, using the same readback format. One of them is going to be incorrect.

Reply:
Yes normal, like other already set bgra for both

Reply:
Same as previous. If both are working you have serious problems somewhere else.